pkg/vault/contracts/CompositeLiquidityRouter.sol

```diff
@@ -2,8 +2,8 @@
 
 pragma solidity ^0.8.24;
 
-import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import { IPermit2 } from "permit2/src/interfaces/IPermit2.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
Suggest restoring this. Minor style point, but for main (i.e., not test) contracts, I prefer organizing them in descending length order for readability.